src/Console/Command/FixCommand.php

```diff
@@ -174,6 +174,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $progressType = $resolver->getProgress();
         $finder = $resolver->getFinder();
 
+        if (null !== $stdErr && $resolver->configFinderIsOverridden()) {
+            $stdErr->writeln(
+                sprintf($stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s', 'Paths from configuration file have been overridden by paths provided as command arguments.')
```

Comment:
added `->isDecorated` for now for consistency.
we need to analyse do we still need to check `->isDecorated` on our side (and maybe drop it globally, not only for this one message)